package.json

```diff
@@ -2,16 +2,22 @@
   "name": "test_task_5",
   "version": "0.1.0",
   "private": true,
+  "homepage": "https://a-lukianenko.github.io/remake",
   "dependencies": {
     "@reduxjs/toolkit": "^1.5.0",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "classnames": "^2.2.6",
     "date-fns": "^2.17.0",
     "formik": "^2.2.6",
+    "localbase": "^0.7.4",
+    "lodash.isequal": "^4.5.0",
+    "ramda": "^0.27.1",
```

Comment:
**localBase** facilitates working with IndexedDB. I asked you in Slack if I could use it.
**lodash.isequal** compares objects (since I write to the Storage on each unmount and beforeunload I need a way to know whether the user starts filling out a new form or wants to continue where we left off. I use lodash.isequal to compare Storage values with the initialFormValues. If they are equal - no need to show "Do you want to continue" modal.  
(this is important bcs I write to Storage on each unmount and beforeunload. The user can navigate to /add-user, do nothing, navigate to /users and then back to /add-user - and will see the pop-up even though he didn't touch the form )
**rambda** - forgot to uninstall